custom_components/tuya_local/devices/casdon_td_pro_2.yaml

```diff
@@ -0,0 +1,206 @@
+name: Casdon TD Pro 2
+products:
+  - id: wab6oustdukujpkn
+    name: 蒸烤箱TD-pro二代
+primary_entity:
+  entity: switch
+  icon: "mdi:power"
+  name: Switch
+  dps:
+    - id: 1
+      type: boolean
+      name: switch
+
+secondary_entities:
+  - entity: switch
+    name: Start
+    icon: "mdi:play-pause"
+    dps:
+      - id: 2
+        name: switch
+        type: boolean
+  - entity: number
+    name: cook_temperature
+    dps:
+      - id: 7
+        name: value
+        type: integer
+        unit: C
+        range:
+          min: 0
+          max: 500
+  - entity: sensor
+    name: temp_current
+    class: temperature
+    dps:
+      - id: 8
+        name: sensor
+        type: integer
+        unit: C
+        readonly: true
+        range:
+          min: 0
+          max: 500
+  - entity: number
+    name: appointment_time
+    dps:
+      - id: 9
+        name: value
+        type: integer
+        optional: true
+        range:
+          min: 0
+          max: 1440
+        unit: min
+        class: measurement
+  - entity: number
+    icon: "mdi:timer-edit"
+    name: cook_time
+    dps:
+      - id: 10
+        name: value
+        type: integer
+        range:
+          min: 0
+          max: 1440
+        unit: min
+        class: measurement
+  - entity: sensor
+    icon: "mdi:timer"
+    name: remain_time
+    dps:
+      - id: 11
+        name: sensor
+        type: integer
+        readonly: true
+        range:
+          min: 0
+          max: 1440
+        unit: min
+        class: measurement
+  - entity: sensor
+    name: fault_message
+    category: diagnostic
+    class: enum
+    dps:
+      - id: 13
+        type: bitfield
+        name: sensor
+        mapping:
+          - value: OK
+          - dps_val: 0
+            value: OK
+          - dps_val: 1
+            value: "er1：腔体高温保护(腔体传感器温度高于290°C)"
```

Comment:
Again, unless this is really only Chinese market only, these would be better translated to English so more people can read them.  Maybe this does not need to be a sensor, but can instead be an extra attribute on the problem binary_sensor below.

Reply:
I have now translated these descriptions into English, but I would like to have versions in Chinese as well. This product is sold only in China.